tests/transformer/operators/test_arithmetic_operators.py

```diff
@@ -0,0 +1,125 @@
+from RestrictedPython._compat import IS_PY35_OR_GREATER
+from tests import c_eval
+from tests import e_eval
+
+import pytest
+
+
+# Arithmetic Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+@pytest.mark.parametrize(*e_eval)
+def test_Add(c_eval, e_eval):
```

Comment:
Sorry, that I was not clear enough: I think it is enough to test using `e_eval` for nodes which are allowed without restrictions. The tests for these nodes can drop the code using `c_eval`.

So the only line for this test would be:

```
assert e_eval('1 + 1') == 2
```

The rest is already tested elsewhere.